tests/models/speech_to_text/test_modeling_speech_to_text.py

```diff
@@ -701,52 +701,38 @@ def test_tf_from_pt_safetensors(self):
 @require_torchaudio
 @require_sentencepiece
 @require_tokenizers
-@slow
 class Speech2TextModelIntegrationTests(unittest.TestCase):
-    @cached_property
-    def default_processor(self):
-        return Speech2TextProcessor.from_pretrained("facebook/s2t-small-librispeech-asr")
-
-    def _load_datasamples(self, num_samples):
-        from datasets import load_dataset
-
+    @classmethod
+    def setUpClass(cls):
```

Comment:
prevents regressions: the model here is super small, these 2 tests take <10s on CPU

most of the time is in loading the model and dataset, having `setUpClass` cuts the run time from 15 to 10s